signature/signature.go

```diff
@@ -165,12 +173,13 @@ func Verify(data []byte, sig []byte, privateKeyURI string) (bool, error) {
 // LoadKeyringPairFromEnv looks up whether the env variable TEST_PRIV_KEY is set and is not empty and tries to use its
 // content as a private phrase, seed or URI to derive a key ring pair. Panics if the private phrase, seed or URI is
 // not valid or the keyring pair cannot be derived
-func LoadKeyringPairFromEnv() (kp KeyringPair, ok bool) {
+// Can take a Network property, leave empty for default
+func LoadKeyringPairFromEnv(network string) (kp KeyringPair, ok bool) {
```

Comment:
What about loading the network also from an env variable here?

Reply:
Yep, I can do that